about.css

```diff
@@ -0,0 +1,34 @@
+p{
+    padding:20px 0px;
+}
+h1{
+font-size: 30px;
+}
+.header{
+    font-family: Luminari;
+    color:blue
+}
+*>li{
```

Comment:
No need to include the * and >. Using the li tag alone will select all lis on the page